setup.cfg

```diff
@@ -1,2 +1,9 @@
 [aliases]
-test=pytest
\ No newline at end of file
+test=pytest
+
+[tool:pytest]
+addopts=--flake8
+
+[flake8]
+max-line-length=120
+ignore=W605
```

Comment:
I see that flake 8 is ignoring invalid escape sequences (https://lintlyci.github.io/Flake8Rules/rules/W605.html), did a build fail due to this? @demianbrecht